### PR TITLE
Update sample manifests for KubeOne cluster creation

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
We should, right from the start, instruct our users to use external CCM whenever it's available.

### CODE REVIEW

**Comment by kron4eg** [comment]
/lgtm

@xmudrii PTAL

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md [hunk 1]
```diff
@@ -861,9 +875,10 @@ automatically by KubeOne as of v1.0.4.**
 apiVersion: kubeone.k8c.io/v1beta2
 kind: KubeOneCluster
 versions:
-  kubernetes: '1.22.5'
+  kubernetes: '1.25.6'
 cloudProvider:
   vsphere: {}
+  external: true
```
Comment (xmudrii) [inline content/kubeone/main/tutorials/creating-clusters/_index.en.md:881]: For vSphere we also need `csiConfig` or otherwise validation is going to fail.

**Review by xmudrii — CHANGES_REQUESTED** [review]

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md (hunk 1 above)
Comment (ahmedwaleedmalik) [reply content/kubeone/main/tutorials/creating-clusters/_index.en.md:881]: Good catch, that's actually very important.

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md [hunk 2]
```diff
@@ -885,6 +900,17 @@ cloudProvider:
 
     [Network]
     public-network = "NAT Network"
+  csiconfig: |
```
Comment (xmudrii) [inline content/kubeone/main/tutorials/creating-clusters/_index.en.md:903]: It should be `csiConfig`:

```suggestion
  csiConfig: |
```

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md [hunk 3]
```diff
@@ -885,6 +900,17 @@ cloudProvider:
...
+    [Global]
+    cluster-id = "<CLUSTER-ID>"
+    cluster-distribution = "<CLUSTER-DISTRIBUTION>"
```
Comment (xmudrii) [inline content/kubeone/main/tutorials/creating-clusters/_index.en.md:906]: Can we document how to set those two values? The first value should be the name of the cluster, but I'm not sure about the second value.

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md [hunk 4]
```diff
@@ -885,6 +900,17 @@ cloudProvider:
 
     [Network]
     public-network = "NAT Network"
+  csiconfig: |
+    [Global]
```
Comment (xmudrii) [inline content/kubeone/main/tutorials/creating-clusters/_index.en.md:904]: Don't we need credentials here as well? We have credentials here in E2E tests for example. Also, do we need parameters such as default-datastore and folder?

**Review by xmudrii — CHANGES_REQUESTED** [review]
Some more comments related to vSphere.

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md (hunk 3 above)
Comment (ahmedwaleedmalik) [reply content/kubeone/main/tutorials/creating-clusters/_index.en.md:906]: I basically picked this up from our kubeone init docs :D But let me take a look at this stuff.

File: content/kubeone/main/tutorials/creating-clusters/_index.en.md (hunk 4 above)
Comment (ahmedwaleedmalik) [reply content/kubeone/main/tutorials/creating-clusters/_index.en.md:904]: Credentials are already there. Folder and datastore are something that we use but are not a requirement. 

The configuration in the documentation looks fine now. We may want to document other options in the CSI config but that's out of scope for this PR.

**Review by xmudrii — APPROVED** [review]
Thank you!
/lgtm
/approve

**Pull request merged**